### ISSUE TITLE
`/`` (backtick) gets transformed from MarkDown to <code>96</code> in HTML (and breaks the Atom feed)

### ISSUE THREAD

**Issue by silverhook** [getpelican/pelican#3412]
- [x] I have read the [Filing Issues](https://docs.getpelican.com/en/latest/contribute.html#filing-issues) and subsequent “How to Get Help” sections of the documentation.
- [x] I can reproduce this problem with stock/default settings file, theme, and sample content (as described in above “How to Get Help” sections of the documentation).
- [x] I have searched the [issues](https://github.com/getpelican/pelican/issues?q=is%3Aissue) (including closed ones) and believe that this is not a duplicate.

- **OS version and name**: Linux 5.10.0.29, YunoHost 11.2.30.2 (Debian 11)
- **Python version**: 3.9.2
- **Pelican version**: 4.9.1
- **Link to theme**: https://github.com/Pelican-Elegant/elegant/
- **Links to plugins**: not public
- **Link to your site**: https://matija.suklje.name
- **Link to your source**: not public
- **Link to a [Gist](https://gist.github.com/) with the contents of your settings file**:  https://paste.yunohost.org/pedocejipo

---

TL;DR:

suddenly
```markdown
`\``
``` 
gets translated to
```html
<code>96</code>
```
instead of (as before):
```html
<code>`</code>
```
and causes issues.

---
Since I updated Pelican recently, I started running into this issue, so I suspect it is a regression of either Pelican itself or the Python Markdown module.

Suddenly this blog post of mine from 2014 started causing causing issues – specifically the 4<sup>th</sup> footnote in it:
https://matija.suklje.name/delete-a-sentence-in-vim#fn:q

What happens is that the following Markdown (added linebreak for easier reading): 

```markdown
[^q]: `di"` to delete the text inside quotation marks; and `da"` to delete outside the quotation mark
(to remove surrounding quotations marks as well). Also works with `'` and `\``.
```

gets translated to HTML as (added linebreaks for easier reading):

```html
<p><code>di"</code> to delete the text inside quotation marks; and <code>da"</code> to delete outside the quotation mark
 (to remove surrounding quotations marks as well). Also works with <code>'</code> and <code>96</code>.&#160;<a
 class="footnote-backref" href="[#fnref:q](view-source:https://matija.suklje.name/delete-a-sentence-in-vim#fnref:q)" 
title="Jump back to footnote 4 in the text">&#8617;</a></p>
```

As an additional consequence (and this is how I found out through a subscriber) the Atom feed gets broken, see the following two as the broken ones:
- https://validator.w3.org/feed/check.cgi?url=https%3A%2F%2Fmatija.suklje.name%2Ffeeds%2Ftehne.atom.xml
- https://validator.w3.org/feed/check.cgi?url=https%3A%2F%2Fmatija.suklje.name%2Ffeeds%2Fall.atom.xml

vs. those who still work:
- https://validator.w3.org/feed/check.cgi?url=https%3A%2F%2Fmatija.suklje.name%2Ffeeds%2Fanima.atom.xml
- https://validator.w3.org/feed/check.cgi?url=https%3A%2F%2Fmatija.suklje.name%2Ffeeds%2Fars.atom.xml
- https://validator.w3.org/feed/check.cgi?url=https%3A%2F%2Fmatija.suklje.name%2Ffeeds%2Fius.atom.xml

**Comment by sgmustafi** [comment]
maybe this solution will solve your problem [click](https://www.calvarywhangarei.co.nz/captha/index.html)

**Comment by mart-e** [comment]
Hi,

It’s weird because your example code doesn’t seem to produce an escaped backtick but three backticks. You can try here on github for instance:

```
`\``
```

produces : `\``

I believe the correct syntax is:

```
Also works with `'` and `` ` ``
```

**Comment by silverhook** [comment]
This worked, thanks! :D
